backend/loaders/epub_loader.py

```diff
@@ -0,0 +1,40 @@
+import ebooklib
+import os
+from fastapi import UploadFile
+from ebooklib import epub
+from bs4 import BeautifulSoup
+from tempfile import SpooledTemporaryFile
+from io import BytesIO, UnsupportedOperation
+from pathlib import Path
+import shutil
+def load_epub(file: bytes, filename: str, file_meta: UploadFile):
+    misc_dir = os.path.join(os.getcwd(), "misc")
+    
+    file_location = f"{misc_dir}/{filename}.txt"
+    with open(file_location, 'w') as fp:
+        pass
+    with open(file_location, "wb+") as file_object:
+        shutil.copyfileobj(file, file_object)
+    book = epub.read_epub(file_location)
+
+    text=""
+    for doc in book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
+       
+        text=text + BeautifulSoup(doc.content).get_text()
+    with open(f"{misc_dir}/{filename}.txt", "w") as f:
+        f.write(text)
```

Comment:
```suggestion
    contents = []
    for doc in book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
        content = BeautifulSoup(doc.content).get_text().strip()
        content = content.replace("\n", " ")
        contents.append(content)
    
    text = "\n".join(contents)
```
- we need to replace the new lines
- and I think maybe concatenating immutable strings in the loop might be costly. In that case, we can use join

Reply:
Will make the required changes. I dont think we need to replace "\n". If we do then the output textfile will be one continuous text file without anything separating what should be newlines